src/main/scala/atom/languageclient/LanguageClientConnection.scala

```diff
@@ -0,0 +1,24 @@
+package laughedelic.atom.languageclient
+
+import io.scalajs.nodejs.events.EventEmitter
+import scala.scalajs.js
+
+// Public: Parameters to send with a workspace/executeCommand request.
+class ExecuteCommandParams (
+  // The identifier of the actual command handler.
+  val command: String,
+  // Arguments that the command should be invoked with.
+  val arguments: js.UndefOr[js.Array[js.Any]] = js.undefined
+) extends js.Object
+
+@js.native 
+trait LanguageClientConnection extends EventEmitter {
```

Comment:
I think this should be a native class, so that it maps on the JS class. See how it's done for `AutoLanguageClient`:

https://github.com/Jarlakxen/scalajs-atom-api/blob/1bbc9c3c7474b64d3f4d01237c83b52db8c8dc98/src/main/scala/atom/languageclient/AutoLanguageClient.scala#L11-L12